tests/config/pytest_modules.yml

```
@@ -3067,9 +3071,9 @@ subworkflows/homer/groseq:
   - subworkflows/nf-core/homer/groseq/**
   - tests/subworkflows/nf-core/homer/groseq/**
 
-subworkflows/vcf_annotate_ensemblvep:
-  - subworkflows/nf-core/vcf_annotate_ensemblvep/**
-  - tests/subworkflows/nf-core/vcf_annotate_ensemblvep/**
+subworkflows/vcf_annotate_ensembl_vep:
+  - subworkflows/nf-core/vcf_annotate_ensembl_vep/**
+  - tests/subworkflows/nf-core/vcf_annotate_ensembl_vep/**
```

not sure what happened here, did you adapt this in any way?

---

I might have messed up the merge conflicts at some point for this one 🫢!

---

ok probably merging master into it should fix it. Otherwise you could revert it manually. Although, changing the name should trigger some tests I thought @maxulysse @mirpedrol? Or would there be a way to prevent this happening, seems dangerous :D

---

That's probably due to some conflicts, yes. There aren't specific tests for name changes, but it is actually dangerous :rotating_light:

---

How do we resolve this issue though?

---

```suggestion
subworkflows/vcf_annotate_ensemblvep:
  - subworkflows/nf-core/vcf_annotate_ensemblvep/**
  - tests/subworkflows/nf-core/vcf_annotate_ensemblvep/**
```
This one is the current name